### DISCUSSION TITLE
alist挂载115，无法用potplayer 打开

### DISCUSSION THREAD

**Discussion by royalroc-hsp** [AlistGo/alist#7724] [General]
这两天使用alist 挂载的115网盘，一用potplayer 打开就崩溃了。用alist直接在线看没有问题。
附上报错信息

**Comment by lichaochao008** [comment]
我的也是这样 不知道什么原因

**Comment by luoyunliu** [comment]
网页在线看没问题，用potplayer崩溃，用fileball崩溃

failed link: failed get link: readObjectStart: expect { or n, but found <, error found in #1 byte of ...|<!doctypeht|..., bigger context ...|<!doctypehtml><html lang="zh-cn"><meta charset="utf|...

**Reply by royalroc-hsp** [reply]
问题一样

**Reply by luoyunliu** [reply]
alist挂载115，用alist-strm生成strm流文件到本地，然后用emby生成海报墙，目前用fileball播放正常。
真是各种绕弯，麻烦的要命，但能看

**Reply by luoyunliu** [reply]
> alist挂载115，用alist-strm生成strm流文件到本地，然后用emby生成海报墙，目前用fileball播放正常。 真是各种绕弯，麻烦的要命，但能看

一开始好的，看了几集之后挂了

**Reply by royalroc-hsp** [reply]
换用cd2吧，我试了，cd2 可以看

**Comment by lengbone** [comment]
我也遇到了该问题

**Comment by tianyun363** [comment]
115 alist 源设为网页版就可以用potplayer

**Reply by royalroc-hsp** [reply]
回头试试

**Reply by AYIE5** [reply]
源设置为网页版还是不能播放

**Comment by Kikyo66** [comment]
啊啊啊我也是这个问题，而且本来网页可以正常播放的，一旦potplayer试图访问之后，网页端就也报错打不开了

**Comment by ashinbao** [comment]
我也是这个问题 infues alist挂115 看一集就报错 什么原因呀 看一集要换一次token
failed link: failed get link: readObjectStart: expect { or n, but found <, error found in #1 byte of ...|<!doctypeht|..., bigger context ...|<!doctypehtml><html lang="zh-cn"><meta charset="utf|...
